.github/workflows/ci.yaml

```diff
@@ -189,6 +189,10 @@ jobs:
           inv build
           inv dev-env -b frr --enable-webhooks
 
+      - name: Apply patch
```

Comment:
Can you add a comment here on why we need this?

Reply:
sure